File: JavaScript/exercicio16.js
function macas(){
    let maca = document.querySelector("#macas").value
    let mostra = document.getElementById('mostra')

    if(maca < 12){
        preco = Number(maca) * 1.30
        mostra.innerHTML = 'O valor total a ser pago pelas maças R$'+preco
    } else{
        preco = Number(maca) * 1.00
        mostra.innerHTML = 'O valor total a ser pago pelas maças R$'+preco
    }
}

File: Java/faccat/Exercicio16.java
package faccat;

import java.util.Scanner;

public class Exercicio16 {
	public static void main(String[] args) {
		Scanner ler = new Scanner(System.in);

        int maca;
        double preco;

        System.out.println("Digite quantas maças foram compradas:");
        maca = ler.nextInt();

        if(maca < 6){
            preco = maca * 1.30;
            System.out.printf("O total a ser pago pelas %d maças compradas é R$%.2f",maca,preco);
        } else{  
            preco = maca * 1.00;
            System.out.printf("O total a ser pago pelas %d maças compradas é R$%.2f",maca,preco);
        }
        
        ler.close();
	}
}
